### PR TITLE
Add Apprentice Curriculum

### PR DESCRIPTION
**Pull request by gtempus**
This is the first of a few steps to migrate the apprenticeship curriculum into the standard.

# Standard Structure
Your audience:
- New hires and current Sparkbox employees
- New clients via proposal documents

Unlike Foundry articles, these contributions are being created directly in Github, not in google docs.

**The goal for this first effort is to:** 
- [x] Add a section in the standard about the apprenticeship 
- [x] Add a paragraph or two that explains the program and introduces the curriculum we already have available publicly.
- [x] Link to the publicly available curriculum

The general structure we’re looking to create is **what→ why→ how**. This means it will start with what our process is from a high-level view. Then it will move on to the rationale for why this topic is important to Sparkbox and why we have come to the conclusions we have about it. Then it will dive down into a more detailed, task-based description of what we do, likely using bullet points or numbered lists.

# Writing Guidelines
These are some general writing guidelines to establish a consistent voice in the standard. Of course, I’ll be editing and ensuring final drafts sound like this, so don’t agonize over these. If you have any questions, shoot them my way!

Write in plural first person about Sparkbox. Refer to Sparkbox as “we” (example: We established this policy to create design consistency. Sparkbox follows these guidelines.)
Do NOT write in singular first person (bad example: In my job, I do...)
Do NOT write in second person (bad example: You should follow these guidelines.)
Write current policies and all tasks in present tense (example: We follow three guidelines. Click on the pencil icon.)
Write specific tasks as imperative actions without a subject 
(example: 1. Click on the lock icon. 2. Type the password. 3. Click log in)

### CODE REVIEW

File: culture/knowledge-sharing/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Knowledge Sharing
 
 - **[Foundry](../../foundry)**
-- **Apprenticeships**
+- **[Apprenticeships](../../apprenticeship)**
```
Comment (catheraaine) [inline culture/knowledge-sharing/README.md:4]: ohhh nice that you added this link here, too!

File: apprenticeship/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Apprenticeship
+Our vision for our apprenticeship program is to produce advanced beginners who are ready to contribute on real, team web projects.
+
+ We are a team that values education. We offer paid apprenticeships because we love what we do, we want to help create new talent, and we genuinely care about the future of the web.
+
+ In the first months of the apprenticeship, apprentinces learn our convictions on priority of content, value of design, semantic HTML, and mobile-first CSS. Apprentices are exposed to our Github-centric project management, deployment processes, best practices for clean code and separation of concerns, and unit testing techniques.
```
Comment (catheraaine) [inline apprenticeship/README.md:6]: spelling on `apprentices` here

File: apprenticeship/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Apprenticeship
```
Comment (catheraaine) [inline apprenticeship/README.md:1]: my markdown linter wants a newline after the title

File: apprenticeship/README.md (hunk 2 above)
Comment (gtempus) [reply apprenticeship/README.md:6]: Doh! Good catch

**Comment by catheraaine** [comment]
Looks great, @gtempus !

**Pull request merged**